spec/cypress/support/on-rails.js

```diff
@@ -14,8 +14,16 @@ Cypress.Commands.add("appCommands", (body) => {
 
 let createdRecords = {};
 export default {
-  getCreatedRecord(factory, index = 0) {
-    return createdRecords[factory] && createdRecords[factory][index];
+  getCreatedRecord(factory, givenIndex = -1) {
+    if (!createdRecords[factory]) {
+      return null;
+    }
+
+    let index = givenIndex;
+    if (index < 0) {
+      index = createdRecords[factory].length + index;
+    }
+    return createdRecords[factory][index];
```

Comment:
Huh, what is this for?

Reply:
That is a bit that handles steps like "school was created with created cohort". By default it should take the most recently created record, but it was always taking the first one - which was causing some odd failures.

There's a much bigger bug there to fix though - for some reason, the `createdRecords` is not being reset between the tests (or rather between some tests). This need more debugging and this seems to be fixing the issue for that PR.